README.md

```diff
@@ -1,2 +1,25 @@
 # moleculer-mongo-checker
-Mongo checker functions for r2d2bzh/moleculer-healthcheck-middleware
+
+This project provides health checkers for [moleculer-healthcheck-middleware](https://github.com/r2d2bzh/moleculer-healthcheck-middleware) for project using [moleculer-db-adapter-mongo](https://github.com/moleculerjs/moleculer-db/tree/master/packages/moleculer-db-adapter-mongo).
+
+
+## Usage
+
+This project gives createChecker functions to use with [moleculer-healthcheck-middleware](https://github.com/r2d2bzh/moleculer-healthcheck-middleware).
+
+**Initialize the Mongo Liveness check**
+
+```js
+const HealthMiddleware = require('@r2d2bzh/moleculer-healthcheck-middleware');
+const HealthcheckMongo = require('moleculer-mongo-healthcheck');
```

Comment:
Do we already want to document the fact that the library will be available on the public npm repo ?

```js
const HealthcheckMongo = require('@r2d2bzh/moleculer-mongo-healthcheck');
```

Reply:
Fixed in https://github.com/Vivien-lelouette/moleculer-mongo-checker/commit/83bff8ae4d01e5734d7d32238d88b092964e7e1c